core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -73,13 +73,15 @@ class JsonProtocolSuite extends SparkFunSuite {
       BlockManagerId("Scarce", "to be counted...", 100))
     val unpersistRdd = SparkListenerUnpersistRDD(12345)
     val logUrlMap = Map("stderr" -> "mystderr", "stdout" -> "mystdout").toMap
+    val workerUrlMap = Map("url" -> "spark://Worker@192.168.1.104:32790",
+      "ui_url" -> "http://192.168.1.104:46445").toMap
```

Comment:
I feel like these ip's should be something more generic, maybe check if there any other places in the test code that have strings with ip and ports that you could copy? Not an actual problem though since whats in the string only has to match the string below.